includes/definitions/enexus.yaml

```diff
@@ -6,30 +6,17 @@ nobulk: 1
 mib_dir:
     - eltek
 discovery:
-    -
-      snmpget:
+    - snmpget:
           oid: .1.3.6.1.4.1.12148.10.2.6.0
-          value:
-            - 'Micropack System'
+          value: 'Micropack System'
 poller_modules:
-    applications: 0
-    bgp-peers: 0
```

Comment:
How come these have been re-enabled. If the devices don't support them then no point in polling/discoverying them.

Reply:
Because there is no point in disabling them, and I hate people copying things like this.

It saves one sql query to disable them.

Reply:
Also, services isn't poller/discovery module.  The wireless module doesn't even do an sql query if there is no support.